### PR TITLE
Adding Almeirim Stone Soup recipe

### PR DESCRIPTION
**Pull request by lisbonjoker**
Includes the famous tale of the stone soup, I can remove if it's too much.

### CODE REVIEW

File: src/almeirim-stone-soup.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+- In a saucepan, boil the kidney beans with the pig's ear, sausages, pork belly, onions, garlic and bay leaf in some water and add a well-washed stone. Season with the olive oil and salt and pepper to taste.
+- If necessary, add more water while it’s boiling. 
+- Once the meat is cooked, take it out and reserve, then add the diced potatoes and cilantro to the pot.
+- Let the potatoes cook on medium to high heat for about 30-35 minutes.
+- Once the potatoes are well cooked, you can remove the pot from the heat, add the previously chopped meat back in and stir it well.
+- Pour into bowls and serve.
+
+## Contribution
+
+- Artur Mancha -- [Pleroma](https://pleroma.pt/@lisbonjoker
```
Comment (LukeGoule) [inline src/almeirim-stone-soup.md:34]: Missing end-bracket.

Comment (lisbonjoker) [reply src/almeirim-stone-soup.md:34]: Oh damn nice catch, thank you

**Comment by LukeSmithxyz** [comment]
Well, I might prefer the extra info at the bottom, but I'll let it pass now.

**Comment by develop1st** [comment]
Not even one day in and there's a recipe that required me to scroll down over a story just to get to the ingredients. Sad!

**Pull request merged**